src/test/incremental/thinlto/cgu_keeps_identical_fn.rs

```diff
@@ -3,29 +3,20 @@
 // ends up with any spans in its LLVM bitecode, so LLVM is able to skip
 // re-building any modules which import 'inlined_fn'
 
-// revisions: cfail1 cfail2 cfail3 cfail4 cfail5 cfail6
-// [cfail4]compile-flags: -Zincremental-relative-spans
-// [cfail5]compile-flags: -Zincremental-relative-spans
-// [cfail6]compile-flags: -Zincremental-relative-spans
+// revisions: cfail1 cfail2 cfail3
```

Comment:
The test diffs are unreadable.
Perhaps it's better to remove non-default revisions in one commit, then flip the switch, and update the tests for the new default behavior in another commit.

Reply:
I think that removing the `-Zincremental-ignore-spans` option is a bad idea. Most of the test fallout came from there. The commits are much simpler now.